tox.ini

```diff
@@ -11,12 +11,14 @@ skipdist = True
 setenv =
        PYTHONPATH = {toxinidir}
 
-deps = -r{toxinidir}/test-requirements.txt
+# The long URL is temporary; when protobuf can successfully install from egg again,
```

Comment:
There's a new version: https://pypi.python.org/pypi/protobuf/3.2.0rc1.post1

If that works, let's remove this.